### PR TITLE
Updated documentations (Part 1)

### PR DESCRIPTION
**Pull request by ijlee2**
## Background

@amyrlam, @jaredgalanis, and I met to learn the steps for publishing an issue of The Ember Times.

From the meeting, we saw the need to maintain an updated, clear set of instructions for:

- How to write for The Ember Times, for future writers
- How to publish the issue, for future maintainers

## Description

I plan to make 2, maybe 3, additional PRs to address #752.

In the 1st PR, I broke down the long sets of instructions in `README.md` and `source/CONTRIBUTING.md` by making several files in the `documentations` directory (cut-paste).

I also updated some external links.

## How to review

Since the only files that are changed are Markdown files, I think a good way to review changes is to click the `Files changed`, then the `View file` button for `README.md` ("entry point").

From `README.md`, you will be able to visit `CONTRIBUTING.md`. From the latter, to a couple of documentation files related to running the app locally.

You can also check commits one by one.

### CODE REVIEW

File: README.md
Comment (ijlee2) [inline README.md:15]: These repos are archived now.

File: CONTRIBUTING.md
Comment (ijlee2) [inline CONTRIBUTING.md:5]: I took some of the writing that used to be in `README.md`.

**Review by amyrlam — COMMENTED** [review]
looks good, thanks for doing this! i would suggest wait for a few others to reply before merging in, which is why i marked my "approval" as a comment

i would prob keep the "how to run with docker or middleman" instructions in the README, cause that's where they usually are in other repos. 

context: we've had some people complain that the contributing instructions are NOT in the README. like, i'm fine with having a super long README, so that it's less confusing. happy to make a commit for this - no strong opinion though! everything will be changing eventually

**Comment by ijlee2** [comment]
Yep, totally okay with getting more feedback before merging the PR.

I think it may be better to create a dedicated `CONTRIBUTING.md` file. It's one of GitHub Community page's recommendations, and would help the `README.md` be kept short.

Community page shows its recommendation for creating CONTRIBUTING.md


I think running the app locally isn't something that many people (I guess I think of The Ember Times' regular or guest writers as the usual visitors to this repo) would do.

But, if they do, I'm hoping that they can readily find the instructions by clicking on the link to `CONTRIBUTING.md`. There, they'd find the section `Writing for The Ember Times` or `Running the app locally`. (The `CONTRIBUTING.md` is currently unfinished so there may be rearranging of contents later.)

**Comment by amyrlam** [comment]
makes a lot of sense - sgtm!

File: CONTRIBUTING.md
Comment (jaredgalanis) [inline CONTRIBUTING.md:26]: I think the code blocks broke the numbering scheme.

File: CONTRIBUTING.md
Comment (jaredgalanis) [inline CONTRIBUTING.md:40]: Also here.

**Review by jaredgalanis — DISMISSED** [review]
Thanks for getting this started! One small thing re the numbering.

File: CONTRIBUTING.md
Comment (ijlee2) [reply CONTRIBUTING.md:26]: ~Ah. GitHub knows to auto-increment the numbers. Using `1` always lets us skip manually changing the numbers should the list need to change.~

Never mind. The preview does show all 1's. Weird.

File: CONTRIBUTING.md
Comment (ijlee2) [inline CONTRIBUTING.md]:
```suggestion
    ```bash
    git clone git@github.com:<your GitHub handle>/ember-blog.git
    ```
```

File: CONTRIBUTING.md
Comment (ijlee2) [inline CONTRIBUTING.md]:
```suggestion
    ```bash
    cd ember-blog
    ```
```

File: CONTRIBUTING.md
Comment (ijlee2) [inline CONTRIBUTING.md]:
```suggestion
    ```bash
    # Create an Ember Times blog post for August 29, 2020
    touch source/2020-08-29-the-ember-times-issue-162.md
    ```
```

File: CONTRIBUTING.md
Comment (ijlee2) [reply CONTRIBUTING.md:26]: It looks like I had to use 4 spaces instead of 2 for the preceding code blocks.

**Review by amyrlam — APPROVED** [review]
let's merge this in and iterate if needed, hope that's cool! i want to add a note from friday and will make a new branch

**Pull request merged**
